keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueInputSplit.java

```diff
@@ -42,6 +44,14 @@
   private long mBlockId;
 
   /**
+   * Default constructor, to be used in de-serialization of {@link KeyValueInputSplit}.
```

Comment:
Why isn't it a problem that the deserialized input split has an invalid block id?